app/services/idv/steps/doc_auth_base_step.rb

```diff
@@ -87,7 +90,10 @@ def throttled_url
       end
 
       def throttled_else_increment
-        Throttler::IsThrottledElseIncrement.call(user_id, :idv_acuant)
+        Throttle.for(
+          target: user_id,
```

Comment:
Should this have been... ?

```suggestion
          user: user_id,
```

Reply:
🤦  I think it should have been:

```suggestion
          target: current_user,
```

but yes